doc/plugins/prometheus.md

```diff
@@ -120,6 +120,10 @@ Or you can goto [Grafana official](https://grafana.com/grafana/dashboards/11719)
 
 ![](../../doc/images/plugin/grafana_3.png)
 
+![](../../doc/images/plugin/grafana_4.png)
```

Comment:
![Uploading image.png…]()

Reply:
There are currently two problems for histogram:
1. The legends have leading for trailing zeros ([reason](https://github.com/knyar/nginx-lua-prometheus/blob/499de49cab0217c855dd9be12c3c1596cc2e9827/prometheus.lua#L24-L27)) so it looks messy
2. There are two many buckets out there. We don't need buckets like "30,40,50,60,70,80,90,100,200,300,400,500". After reduce buckets, it somewhat looks better
![image](https://user-images.githubusercontent.com/34589752/106387869-c3162a80-6416-11eb-979f-aeada2f7564b.png)

Or we could use 95-percentile as @tokers said. I'm not so familiar with monitor APISIX, so what do you think about that

Reply:
For `etcd-reachable` and `nginx metric errors`, the comments of what these two metrics do are explained on the metrics page alongside with the data. And for how I implement it
- `etcd-reachable`: when etcd is reachable, the metric would be 1 and the graph would be green. While it is disconnected, the metric turns to 0 and the panel would be red.
- `nginx metric errors`: it measures how many errors of nginx-lua-prometheus. I also leave it to only show the current error number without time series, since if it is larger than 0, it should be taken care of. And also if it is larger than 0, it would turn red.

Reply:
would fix leading for trailing zeros in https://github.com/knyar/nginx-lua-prometheus/pull/119

Reply:
Hi @membphis PTAL

Updated to newer version of nginx-lua-prometheus, and it would looks like
![image](https://github.com/apache/apisix/blob/5a45ba741522d275e83eeaa4bfcd01f09e69e148/doc/images/plugin/grafana_4.png?raw=true)